packages/dappmanager/src/utils/toBytes.ts

```diff
@@ -0,0 +1,24 @@
+import { ToBytes } from "../types";
+
+/**
+ * Function reuturns bytes. Accepts mb, kb and gb
+ * @param param0
+ */
+export function toBytes({
```

Comment:
You can use as signature
```ts
toBytes(amount: number, unit: "kb" | "mb" | "gb")
```
Not sure it's necessary to nest the options and the amount and unit types do not overlap so it's "safe" to use normal args

Reply:
done